File: Basic Array/Reverse an array.py
"""
📝 Question:
Given an array, reverse the order of its elements and return the reversed array.

Example:
  Input:  [1, 2, 3, 4, 5]
  Output: [5, 4, 3, 2, 1]
"""

# Reverse an Array

def reverse_array(arr):
    return arr[::-1]


if __name__ == "__main__":
    arr = [1, 2, 3, 4, 5]
    print(f"Original: {arr}")
    print(f"Reversed: {reverse_array(arr)}")

File: Basic Array/Reverse an array.cpp
class Solution{
public:
    void reverse(int arr[], int n){
        int temp[n];
		for(int i = 0; i <=n-1; i++)
		{
			temp[n-i-1] = arr[i];
		}
		for(int i= 0; i <= n-1; i++)
		{
			arr[i] = temp[i];
		}
        
        
    }
};
